extension.ts

```diff
@@ -407,6 +408,18 @@ export async function activate(context: vscode.ExtensionContext) {
     toggleExtension(configuration.disableExtension, compositionState);
   });
 
+  registerCommand(context, 'vim.vimTutor', async () => {
+    const setting: vscode.Uri = vscode.Uri.parse('untitled:' + 'vimtutor.txt');
```

Comment:
Can you explain the `untitled:`?

Reply:
Oh, does that tell VSCode the file doesn't exist on disk yet? Is there a reason you can't use VSCode's `workbench.action.files.newUntitledFile` command?

Reply:
If you do need to parse the Uri, could you give `parse` a second argument of `true`? It's a `strict` flag which the docs recommend using.

Reply:
Hey! Actually I used this command because I wanted to target the document and write on it afterwards... I added the `true` as a second argument.